[2024-11-15T22:15:32.252+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-15T22:15:32.296+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: FLOAT_DATA_TO_PG.transform_data manual__2024-11-15T22:15:14.905656+00:00 [queued]>
[2024-11-15T22:15:32.700+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: FLOAT_DATA_TO_PG.transform_data manual__2024-11-15T22:15:14.905656+00:00 [queued]>
[2024-11-15T22:15:32.701+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-15T22:15:32.740+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_data> on 2024-11-15 22:15:14.905656+00:00
[2024-11-15T22:15:32.756+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1575) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-15T22:15:32.761+0000] {standard_task_runner.py:63} INFO - Started process 1577 to run task
[2024-11-15T22:15:32.760+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'FLOAT_DATA_TO_PG', 'transform_data', 'manual__2024-11-15T22:15:14.905656+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/etl_process_float_data.py', '--cfg-path', '/tmp/tmpa_6ktf3q']
[2024-11-15T22:15:32.768+0000] {standard_task_runner.py:91} INFO - Job 52: Subtask transform_data
[2024-11-15T22:15:32.884+0000] {task_command.py:426} INFO - Running <TaskInstance: FLOAT_DATA_TO_PG.transform_data manual__2024-11-15T22:15:14.905656+00:00 [running]> on host 35eb7317174e
[2024-11-15T22:15:33.153+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='FLOAT_DATA_TO_PG' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-15T22:15:14.905656+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-15T22:15:14.905656+00:00'
[2024-11-15T22:15:33.157+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-15T22:15:33.247+0000] {etl_process_float_data.py:52} INFO - Transformed DataFrame:
[2024-11-15T22:15:33.249+0000] {etl_process_float_data.py:53} INFO -      client              project  ...   end_date estimated_hours
0  Client 1  Website Development  ... 2023-07-24             112
1  Client 1  Website Development  ... 2023-07-24              24
2  Client 1  Website Development  ... 2023-08-28             189
3  Client 1  Website Development  ... 2023-09-04              77
4  Client 1  Website Development  ... 2023-09-04              92

[5 rows x 8 columns]
[2024-11-15T22:15:33.298+0000] {xcom.py:675} ERROR - Object of type Timestamp is not JSON serializable. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your *** config or make sure to decorate your object with attr.
[2024-11-15T22:15:33.300+0000] {etl_process_float_data.py:56} ERROR - An error occured: Object of type Timestamp is not JSON serializable
[2024-11-15T22:15:33.301+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-15T22:15:33.301+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-15T22:15:33.316+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=FLOAT_DATA_TO_PG, task_id=transform_data, run_id=manual__2024-11-15T22:15:14.905656+00:00, execution_date=20241115T221514, start_date=20241115T221532, end_date=20241115T221533
[2024-11-15T22:15:33.371+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-15T22:15:33.405+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-15T22:15:33.407+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
